e2e/tests/add-to-cart.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'playwright/test';
+import { COLLECTION_NAME, COLLECTIONS_ENDPOINT, PRODUCT_NAME } from '../constants';
+import { test } from '../fixtures';
+
+test.describe('Add to cart', () => {
+  test.beforeEach('Navigate to the Collections page', async ({ page }) => {
+    test.skip(!COLLECTION_NAME, 'PLAYWRIGHT_COLLECTION_NAME was not defined');
+
+    await page.goto(COLLECTIONS_ENDPOINT + COLLECTION_NAME);
```

Comment:
```suggestion
    test.skip(!COLLECTION_NAME, 'PLAYWRIGHT_COLLECTION_NAME was not defined');
    return;

    await page.goto(COLLECTIONS_ENDPOINT + COLLECTION_NAME);
```

Need to return, otherwise you'll skip but still move forward. Types won't catch this because it's still a string.

Reply:
It sounds weird, we can't proceed if the test was skipped. The type is still `string | undefined`, so if PLAYWRIGHT_COLLECTION_NAME is not set we get the test skipped as expected. I've already checked this out.

Reply:
You're going to this URL if the var is not defined: 

`await page.goto('/collections/undefined');`

I don't think that's what you'd want or intend. I think you should be bailing on the entire suite if this is not defined, we can't perform any collections tests without the collection name.

Reply:
That makes sense to me, but there is no way we can get to `page.goto` if the collection name is not defined. It will not perform `page.goto` after skipping the test.

Reply:
Then you need to return. The function continues to execute as you call `test.skip`. If `page.goto` is going to be a no-op, then it shouldn't be called since it could have side effects or the behavior could change in the future.